_config.yml

```diff
@@ -28,7 +28,7 @@ headlineMain: "Let's Work Together"
 # 2. change 'USER' in the example urls below to the username of the google account you made for your calendar
 # 3. set calendar_on: True.
 
-calendar_on: False
+calendar_on: True
 calendar_embed_url: "https://www.google.com/calendar/embed?src=USER@gmail.com"
```

Comment:
Would you mind removing this file from your pull request? I think you mean to updated your own calendar on your fork, and not the calendar from the studyGroup base repo. Unfortunately, when you submit a pull request, it chains subsequent commits to that PR until your PR is merged. Sorry!